istioctl/pkg/tag/templates/validatingwebhook.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Just curious why we have a copy under istioctl for this, but mutating webhook is using the chart's version? Is this unique from the chart version?

Reply:
Mutating webhooks can move here too since we decided being able to set tags through IstioOperator is not a priority.

Reply:
Will handle in future PR.